cattle/plugins/docker/compute.py

```diff
@@ -192,13 +192,56 @@ def _is_instance_active(self, instance, host):
 
     @staticmethod
     def _get_docker_client(host):
-        cluster_connection = ""
+        cluster_connection = None
+        tls_config = None
         try:
             cluster_connection = host['clusterConnection']
+            if cluster_connection.startswith('https'):
+                try:
+                    account_id = host['accountId']
+                    ca_crt = host['caCrt']
+                    client_crt = host['clientCrt']
+                    client_key = host['clientKey']
+
+                    client_certs_dir = Config.client_certs_dir()
+                    acct_client_cert_dir = \
+                        '{0}/{1}'.format(client_certs_dir, account_id)
+                    if not path.exists(acct_client_cert_dir):
+                        log.debug('Creating client cert directory: %s',
+                                  acct_client_cert_dir)
+                        makedirs(acct_client_cert_dir)
+                    if ca_crt:
+                        log.debug('Writing cert auth')
+                        with open(acct_client_cert_dir + '/ca.crt', 'w') as f:
```

Comment:
Ah, sorry I didn't notice this in the last review, but this should probably use `os.path.join()`